src/Toolkit.Forms/OverviewMap/OverviewMap.cs

```diff
@@ -0,0 +1,219 @@
+                    AttachedView = GeoView,
+                };
+                _overviewMapView.IsAttributionTextVisible = false;
+
+                if (Map == null)
+                {
+                    Map = new Map(BasemapStyle.ArcGISTopographic);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the symbol used to draw the <see cref="GeoView"/>'s visible area.
+        /// </summary>
+        /// <remarks>
+        /// The default is an empty fill symbol with a 1 point red outline.
+        /// </remarks>
+        public Symbol? AreaSymbol
+        {
+            get => GetValue(AreaSymbolProperty) as Symbol;
+            set => SetValue(AreaSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets the geoview whose extent is to be displayed.
+        /// </summary>
+        /// <remarks>
+        /// Note that by default interaction with <see cref="OverviewMap"/> will navigate the attached GeoView.
+        /// </remarks>
+        public GeoView? GeoView
+        {
+            get => GetValue(GeoViewProperty) as GeoView;
+            set => SetValue(GeoViewProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets the Map shown in the inset/overview map.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to a map with a basemap in style <see cref="BasemapStyle.ArcGISTopographic"/>.
+        /// </remarks>
+        public Map? Map
+        {
+            get => GetValue(MapProperty) as Map;
+            set => SetValue(MapProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets the symbol used to draw the <see cref="GeoView"/>'s viewpoint when it isn't possible to show the visible area (for example, when showing a scene).
+        /// </summary>
+        /// <remarks>
+        /// The default is a red cross.
+        /// </remarks>
+        public Symbol? PointSymbol
+        {
+            get => GetValue(PointSymbolProperty) as Symbol;
+            set => SetValue(PointSymbolProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets the amount to scale the overview map's viewpoint compared to the attached <see cref="GeoView"/>.
+        /// </summary>
+        /// <remarks>
+        /// The default is 25.
+        /// </remarks>
+        public double ScaleFactor
+        {
+            get => (double)GetValue(ScaleFactorProperty);
+            set => SetValue(ScaleFactorProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="AreaSymbol"/> bindable property.
+        /// </summary>
+        public static readonly BindableProperty AreaSymbolProperty =
+            BindableProperty.Create(nameof(AreaSymbol), typeof(Symbol), typeof(OverviewMap), new SimpleFillSymbol(SimpleFillSymbolStyle.Null, System.Drawing.Color.Transparent, new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.Red, 1)), propertyChanged: OnAreaSymbolChanged);
```

Comment:
You shouldn't have mutable reference types as default values in dependency properties, since they are static and shared among all. Either specify them in the template or the constructor, and make the default here `null`.